acceptance/tests/resource/user/should_purge.rb

```diff
@@ -0,0 +1,40 @@
+test_name "should purge a user"
```

Comment:
This should be constructed as
```
test_name "should purge a user" do
end
```

and
```
step "Step One" do
end
```

like in e.g. https://github.com/puppetlabs/puppet/blob/5.5.x/acceptance/tests/resource/exec/should_run_command_in_cwd.rb

Reply:
I agree this change would look better. But I'll note that doing this will make this one test different from every test in its directory.

Reply:
I think we have some standardization to do; none of the tests in this `user` directory follow that construction.

Reply:
Some tests do. See e.g.

https://github.com/puppetlabs/puppet/blob/5.5.x/acceptance/tests/resource/user/should_manage_attributes_aix.rb
https://github.com/puppetlabs/puppet/blob/5.5.x/acceptance/tests/resource/user/should_manage_groups.rb

(You might have to rebase the PR with the latest 5.5.x changes). Those are the newer tests though. The convention's definitely the block style I outlined.